setup.py

```diff
@@ -32,6 +32,7 @@
     install_requires=[
         "click>=8",
         "cryptography",
+        "jcs",
```

Comment:
How do we feel about bringing this in as a dependency? The actual PyPI distribution is maintained by someone else but the code is developed here: https://github.com/cyberphone/json-canonicalization

Even if we decide to hand roll this in the future, I think we should use this in the short term to keep things moving as it's probably a decent amount of work to get this right.

Reply:
I'm personally okay with it in terms of keeping things moving, but I agree that we should try to replace it in the medium term.

(We should look into what TUF's codebase does, since I believe they also have to canonicalize JSON for various tasks.)

Reply:
Looks like TUF rolls it's own canonicalizer: https://github.com/theupdateframework/python-tuf/blob/31ca674aa0923e54d7486e5b4a068efb82a34641/tuf/api/serialization/json.py#L90-L104

I'd rather pull in `tuf` as a dependency since we're going to need it eventually anyways.

Reply:
I've pulled in `securesystemslib` since that's what's doing the canonicalisation under the hood. The API that `tuf` exposes with the `Signed` type isn't useful for us so I haven't brought in `tuf`.